client/Dockerfile

```diff
@@ -2,6 +2,19 @@ FROM node:12-slim
 
 WORKDIR /client
 
+RUN apt-get update && \
+    apt-get install -y libgtk2.0-0 \
+    libgtk-3-0 \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    libxtst6 \
+    xauth \
+    xvfb \
+    chromium
+
 RUN npm install -g @vue/cli && \
     npm install -g @vue/cli-service-global && \
```

Comment:
@Aaron-Fine To answer your question from my previous PR.... we're basically doing npm install cause that's how Jacob put it :) I didn't feel like updating it.